experimental/oak_baremetal_channel/schema.fbs

```diff
@@ -3,6 +3,7 @@ attribute "method_id";
 rpc_service TrustedRuntime {
     HandleUserRequest(UserRequest) : UserRequestResponse (method_id: 0);
     Initialize(Initialization) : Empty (method_id: 1);
+    UpdateLookupData(LookupData) : Empty (method_id: 2);
```

Comment:
We should document these methods at some point (either here, or perhaps better, in a markdown file somewhere that we can link from here).

Could you add a comment here for now? Mainly to clarify whether this is a total or partial update (I guess it's total, right?).

Reply:
done